fjord/analytics/analyzer_views.py

```diff
@@ -315,23 +321,25 @@ def unknown_to_empty(text):
         })
 
     if search_source is not None:
-        f &= F(source=search_source)
+        f &= F('term', source=search_source)
         current_search['source'] = search_source
     if search_campaign is not None:
-        f &= F(campaign=search_campaign)
+        f &= F('term', campaign=search_campaign)
         current_search['campaign'] = search_campaign
     search_organic = {'0': False, '1': True}.get(search_organic, None)
     if search_organic in [False, True]:
-        f &= F(organic=search_organic)
+        f &= F('term', organic=search_organic)
         current_search['organic'] = int(search_organic)
 
-    search = search.filter(f).order_by('-created')
+    search = search.filter(f).sort('-created')
 
     # If they're asking for a CSV export, then send them to the export
     # screen.
     if output_format == 'csv':
         return _analytics_search_export(request, search)
 
+    original_search = search._clone()
```

Comment:
I needed to do two sets of aggregation and thought it was easier to do if I cloned it because one set involves an additional filter or something like that. I need to revisit this.